File: GO_Backend_Example1/entities/example1/adm/editables/repoinuse.go
package editables

import (
	"github.com/reizzao/backendgo/entities/example1/repositories"
	types "github.com/reizzao/backendgo/entities/example1/types"
)

type Example1OptionsRepoinuse struct {
	Json   types.RepositoryExample1
	Memory types.RepositoryExample1
}

var optionsRepoinuseExample1 = Example1OptionsRepoinuse{
	Json:   repositories.JsonRepositoryExample1,
	Memory: repositories.MemoryRepositoryExample1,
}

// editable repoinuse # important
var RepoinuseExample1 types.RepositoryExample1 = optionsRepoinuseExample1.Memory

File: Backend_TSDENO_1/src/Domain/entities/Example1/implementations/repoinuse.ts
import { JsonExample1Repository, MemoryExample1Repository } from "@example1";


class Example1RepositoriesOptionsInuse {
  static json() {
    return new MemoryExample1Repository()
  }

  static memory() {
    return new JsonExample1Repository()
  }
}


export { Example1RepositoriesOptionsInuse }
